linera-indexer/plugins/src/operations.rs

```diff
@@ -53,34 +52,37 @@ pub struct ChainOperation {
 }
 
 #[derive(RootView)]
-pub struct OperationsPluginInternal<C> {
+pub struct Operations<C> {
     last: MapView<C, ChainId, OperationKey>,
     count: MapView<C, ChainId, u64>,
     /// ChainOperation MapView indexed by their hash
     operations: MapView<C, OperationKey, ChainOperation>,
 }
 
-#[derive(Clone)]
-pub struct OperationsPlugin<C>(Arc<Mutex<OperationsPluginInternal<C>>>, String);
+#[derive(OneofObject)]
+pub enum OperationKeyKind {
+    Key(OperationKey),
+    Last(ChainId),
+}
 
-impl<C> OperationsPlugin<C>
+/// Implements helper functions on the Rootview
```

Comment:
```suggestion
/// Implements helper functions on the root view.
```

(Or `` `RootView` ``.)